examples/FieldHooks/ComputeFieldHooks/RemoveSymbolsCompute.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { TextField, Sheet, Workbook } from '@flatfile/configure'
+import { SheetTester } from '../../../src/utils/testing/SheetTester'
+
+//This hook checks for and removes special characters in a string using RegEx.
+const removeSymbolsCompute = new Sheet('removeSymbolsCompute', {
```

Comment:
I would rename this to RemoveSymbolsSheet, with a sheetname of `RemoveSymbolsSheet`